app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,4 @@
+class WelcomeController < ApplicationController
+	def index
```

Comment:
Convention for ruby regarding the indentation is 2 spaces. You can set it in sublime settings, tab_size to 2. This should be applied to all ruby files
https://www.sublimetext.com/docs/2/indentation.html